library/lib/javascript.js

```diff
@@ -20,7 +20,7 @@ export function getSharedScriptsForAll(names) {
   return sharedChunks.map(toScripts)
 
   function addChunk ({ dependencies = [], filename }) {
-    dependencies.map(x => manifest[x].filename).forEach(addFilename)
+    dependencies.sort((a, b) => b === 'runtime' ? 1 : 0).map(x => manifest[x].filename).forEach(addFilename)
```

Comment:
This is not good. Please use the property from the manifest. We are not guaranteed it will be called `runtime`.